LoopKit/Service/VersionCheckService.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  VersionCheckService.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 9/8/21.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+// Note: order is important for VersionUpdate.  Later version updates are more critical than earlier ones.  Do not reorder!
+public enum VersionUpdate: Comparable, CaseIterable {
+    /// No version update needed (i.e. running the latest version).
```

Comment:
[nit] No version update needed doesn't mean that it is necessarily running the latest version.